db/migrate/20160312201127_create_leagues.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLeagues < ActiveRecord::Migration
+  def change
+    create_table :leagues do |t|
+      t.string :name
```

Comment:
Let's make sure this attribute is required here and at all the model layer as well.